Assets/_Core/ExoScriptableObject/Scripts/CardObjectInstanciator.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Random = UnityEngine.Random;
+
+public class CardObjectInstanciator : MonoBehaviour
+{
+    private List<string> _cardInInventoryList = new List<string>();
+    
+    private void Start()
+    {
+        
+        for (int i = 0; i < 7; i++)
+        {
+            string randomCard = CreateRandomCard();
+            bool cardInInventory = false;
+            foreach (string card in _cardInInventoryList)
```

Comment:
conseil : afin de bien la distinguer des autres variables, un bon nom de variable d'itération peut se terminer par "Item". On la retrouve plus facilement et on comprend directement son utilité. Ici : `cardItem`